src/python/bot/fuzzers/honggfuzz/engine.py

```diff
@@ -126,19 +155,20 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
 
     crashes = []
     for line in log_lines:
-      crash_match = _CRASH_REGEX.match(line)
-      if not crash_match:
+      reproducer_path = _get_reproducer_path(line)
+      if reproducer_path:
+        crashes.append(
+            engine.Crash(reproducer_path, sanitizer_stacktrace, [],
+                         int(fuzz_result.time_executed)))
         continue
 
-      reproducer_path = crash_match.group(1)
-      crashes.append(
-          engine.Crash(reproducer_path, sanitizer_stacktrace, [],
-                       int(fuzz_result.time_executed)))
-      break
+      stats = _get_stats(line)
+
+    if stats is None:
```

Comment:
you need to initialize stats var too, otherwise can end up with var not defined.

Reply:
Done!